### PR TITLE
Facility user form validations are displayed at the top of the form fields

### LINKED ISSUE #3494: Facility user form validations are displayed at the top of the form fields.
Branch: develop
Expected behavior: facility user form fields validations must show at the bottom of the textbox.
Current behavior: facility user form fields validations show on top of the textbox.

Steps to reproduce:
1. Log in as the administrator 
2. Create a new facility.
3. Leave the fields blank and click create user
screen shot 2015-04-09 at 11 09 15 am
.

### PR DESCRIPTION
**Pull request by amodia**
Hi @aronasorman this fixes #3494 and #3008 

screen shot 2015-04-09 at 11 36 13 am

### CODE REVIEW

File: kalite/facility/templates/facility/facility_user.html [hunk 1]
```diff
@@ -71,7 +71,14 @@ <h3 class="no-margin-top">{{ title }}</h3>
         <form method="post" class="basic-form">
             {% csrf_token %}
 
-            {{ form.as_p }}
+        {% for field in form.visible_fields %}
```
Comment (MCGallaspy) [inline kalite/facility/templates/facility/facility_user.html]: Can you please change this and subsequent lines so that it matches the original alignment?

**Comment by amodia** [comment]
Hi @MCGallaspy done fixing the alignment.

File: kalite/facility/templates/facility/facility_user.html [hunk 2]
```diff
@@ -71,7 +71,14 @@ <h3 class="no-margin-top">{{ title }}</h3>
         <form method="post" class="basic-form">
             {% csrf_token %}
 
-            {{ form.as_p }}
+            {% for field in form.visible_fields %}
```
Comment (MCGallaspy) [inline kalite/facility/templates/facility/facility_user.html]: Test failing presumably because you're not including hidden fields.

Comment (MCGallaspy) [inline kalite/facility/templates/facility/facility_user.html]: Check out the [docs](https://docs.djangoproject.com/en/1.5/topics/forms/#looping-over-hidden-and-visible-fields).

**Comment by amodia** [comment]
@MCGallaspy thanks for your suggestion, now it pass on travis.

**Comment by MCGallaspy** [comment]
Sorry for letting this PR age. Can you just make sure the alignment is again consistent, and also attach another screenshot now that the hidden fields are included, just to verify that they don't actually disrupt the display?

**Comment by amodia** [comment]
@MCGallaspy  this is the new screenshot with hidden fields.

screen shot 2015-04-23 at 9 35 20 am

File: kalite/facility/templates/facility/facility_user.html [hunk 3]
```diff
@@ -71,7 +71,17 @@ <h3 class="no-margin-top">{{ title }}</h3>
         <form method="post" class="basic-form">
             {% csrf_token %}
 
-            {{ form.as_p }}
+            {% for field in form.hidden_fields %}
+                {{ field }}
+            {% endfor %}
+            {% for field in form.visible_fields %}
+               <p>
```
Comment (MCGallaspy) [inline kalite/facility/templates/facility/facility_user.html]: Just use consistent alignment. You've got 4 spaces on line 75 and 3 spaces here. Use 4 spaces everywhere.

**Comment by amodia** [comment]
@MCGallaspy done fixing the alignment.

**Pull request merged**